site/dat/repo/various.json

```diff
@@ -1551,47 +1551,14 @@
       "com.atakama.jmeter.recording.JavaURL"
     ],
     "versions": {
-      "1.0": {
-        "downloadUrl": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0/jmeter-atakama-parameters-1.0.jar",
-		"libs": {
-				"jakarta.activation-api": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0/jakarta.activation-api-1.2.1.jar",
-				"jakarta.xml.bind-api": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0/jakarta.xml.bind-api-2.3.2.jar",
-				"jaxb-core": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0/jaxb-core-2.3.0.1.jar",
-				"jaxb-runtime": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0/jaxb-runtime-2.3.2.jar",
-				"lib-crypt": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0/lib-crypt-1.0.6.jar"
-				
-		},
-        "depends": [
-          "jmeter-core",
-          "jmeter-components"
-        ]
-      },
-	  "1.0.1": {
-		"changes": "Take into account POST data variables ",
-        "downloadUrl": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0.1/jmeter-atakama-parameters-1.0.1.jar",
-		"libs": {
-				"jakarta.activation-api": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0.1/jakarta.activation-api-1.2.1.jar",
-				"jakarta.xml.bind-api": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0.1/jakarta.xml.bind-api-2.3.2.jar",
-				"jaxb-core": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0.1/jaxb-core-2.3.0.1.jar",
-				"jaxb-runtime": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0.1/jaxb-runtime-2.3.2.jar",
-				"lib-crypt": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0.1/lib-crypt-1.0.6.jar"
-				
-		},
-        "depends": [
-          "jmeter-core",
-          "jmeter-components"
-        ]
-      }
-	  ,
-	  "1.0.2": {
-		"changes": "Fix bug when there are parameters with empty values",
-        "downloadUrl": "https://github.com/atakama/jmeter_plugin_variabilization/releases/download/1.0.2/jmeter-atakama-parameters-1.0.2.jar",
+      "1.1.0": {
```

Comment:
I recommend you keeping old version info for your plugin, and add new versions. Look how other plugins do it.

This is important to allow users to downgrade to older version of plugin in case something is broken.

Reply:
Up this new  version, it will be the case. We want the users  to upgrade to this version and start using  the plugin from it

Reply:
We have changed or license system so that's why we have removed the older versions

Reply:
IMO it's a very unfriendly strategy to cause the inconvenience to users. Users should be motivated by value you provide, not the artificial limitations you set. 
That said, I won't insist here, up to you.